spec/requests/api/v1/articles_spec.rb

```diff
@@ -13,8 +14,98 @@
     allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
   end
 
-  describe "GET /api/articles" do
-    before { article }
+  describe "GET /articles" do
+    before do
+      article.update_columns(organization_id: organization.id)
+    end
+
+    # rubocop:disable RSpec/VariableName
```

Comment:
Rubcop has issues with line 102: `let(:Accept) { "application/vnd.forem.api-v1+json" }`, but rswag REQUIRED the case on the `:Accept` symbol in order to correctly set the request header.